WD-Firmware/demo/build/utils.py

```diff
@@ -70,17 +135,54 @@ def fnCheckInstalledDependencis(listDependencis):
     print("unsupported environment, please switch to a linux based machine")
     exit(1)   
     
-def set_outputfiles_names(name, prefix = "", suffix = ""):
+def fnSetOutputFileNames(prefix = ""):
     # format the artifacts files name for the current demo
-    if prefix:
-      name = STR_NAMES_CONCATENATE % (prefix, name)
-    if suffix:
-      name = STR_NAMES_CONCATENATE % (name, suffix)
-    map_file = name + STR_MAP_FILE_EXT
-    elf_file = name + STR_ELF_FILE_EXT
-    return map_file, elf_file
-    
+    #return a list
+    map_file = prefix + STR_MAP_FILE_EXT
+    elf_file = prefix + STR_ELF_FILE_EXT
+    dmp_file = prefix + STR_DMP_FILE_EXT
+    return map_file, elf_file, dmp_file
+
+# set toolchain path
+def fnSetToolchainPath(strTCName, env):
+    if strTCName == STR_TC_LLVM:
+       env['RISCV_LLVM_TC_PATH'] = os.getenv(STR_RV_LLVM_ENV_VAR_NAME)
+       # check if the TC environment variable is set or empty
+       if not env['RISCV_LLVM_TC_PATH']:
+         print ("Error: Set environment variable '" + STR_RV_LLVM_ENV_VAR_NAME + "' to point to the RISCV llvm root")
+         exit(1)
+       # check if the binutils environment variable is set or empty

```

Comment:
can you add one line space here (after exit)

Reply:
done in eccffd7